spec/requests/issues_controller_spec.rb

```diff
@@ -6,6 +6,7 @@
hat only members who continue to meet the requirements over time will remain regulars.)","customfield_10010":null,"customfield_10014":null,"customfield_10015":null,"timetracking":{},"customfield_10005":null,"customfield_10006":null,"customfield_10007":null,"security":null,"customfield_10008":null,"attachment":[],"customfield_10009":null,"aggregatetimeestimate":null,"summary":"[Discourse] Welcome to the Lounge","creator":{"self":"https://example.com/rest/api/2/user?accountId=70121%3A40a02bc7-6a81-4445-9b9e-8d49951a4ebd","accountId":"70121:40a02bc7-6a81-4445-9b9e-8d49951a4ebd","emailAddress":"foo@example.com","avatarUrls":{"48x48":"https://secure.gravatar.com/avatar/93ca0545f57bae0ef29f8545a8389d54?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FDU-6.png","24x24":"https://secure.gravatar.com/avatar/93ca0545f57bae0ef29f8545a8389d54?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FDU-6.png","16x16":"https://secure.gravatar.com/avatar/93ca0545f57bae0ef29f8545a8389d54?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FDU-6.png","32x32":"https://secure.gravatar.com/avatar/93ca0545f57bae0ef29f8545a8389d54?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FDU-6.png"},"displayName":"Foo Bar","active":true,"timeZone":"Europe/Bucharest","accountType":"atlassian"},"subtasks":[],"reporter":{"self":"https://example.com/rest/api/2/user?accountId=70121%3A40a02bc7-6a81-4445-9b9e-8d49951a4ebd","accountId":"70121:40a02bc7-6a81-4445-9b9e-8d49951a4ebd","emailAddress":"foo@example.com","avatarUrls":{"48x48":"https://secure.gravatar.com/avatar/93ca0545f57bae0ef29f8545a8389d54?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FDU-6.png","24x24":"https://secure.gravatar.com/avatar/93ca0545f57bae0ef29f8545a8389d54?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FDU-6.png","16x16":"https://secure.gravatar.com/avatar/93ca0545f57bae0ef29f8545a8389d54?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FDU-6.png","32x32":"https://secure.gravatar.com/avatar/93ca0545f57bae0ef29f8545a8389d54?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FDU-6.png"},"displayName":"Foo Bar","active":true,"timeZone":"Europe/Bucharest","accountType":"atlassian"},"aggregateprogress":{"progress":0,"total":0},"customfield_10000":"{}","customfield_10001":null,"customfield_10002":null,"customfield_10003":null,"customfield_10004":null,"environment":null,"duedate":null,"progress":{"progress":0,"total":0},"votes":{"self":"https://example.com/rest/api/2/issue/DIS-42/votes","votes":0,"hasVoted":false},"comment":{"comments":[],"self":"https://example.com/rest/api/2/issue/10041/comment","maxResults":0,"total":0,"startAt":0},"worklog":{"startAt":0,"maxResults":20,"total":0,"worklogs":[]}}}'
```

Comment:
Might be worth storing this giant JSON string in another file for readability like https://github.com/discourse/discourse/blob/4b2e7eb19ac15951feae275a150c4280225859ec/spec/fixtures/onebox/github_issue_onebox.response